### ISSUE TITLE
lc0AB helper stalling in Cutechess gui on it's 2nd move

### ISSUE THREAD

**Issue by whelanh** [hans-ekbrand/lc0#3]
BUG REPORT

**Describe the bug**
Using the cutechess gui (on Windows 11 system), I find that in a tournament against the conventional lc0 engine, the A/B helper engine will make one move (either as black or white), but then after lc0 makes a move, the A/B helper does not initiate a 2nd move and cutechess reports that it has stalled out.  The A/B move clock in cutechess does not even seem to reset: it seems frozen at the value of the last move.  It appears to me that cutechess cannot restart and/or communicate with the A/B helper engine? (perhaps it's in a wait state?).  I have attached the A/B helper log file as logABengine.txt.  I have also attached the cutechess configuration file for how I configured the A/B helper engine as engines.-.copy.txt.   The A/B engine works fine in Nibbler (so it seems as it's a "restart" issue...not a problem with basic running of the engine).

**Steps to Reproduce**
1.  Initiate a tournament in cutechess gui with 30 seconds per move, selecting lc0 and lcAB as competing engines
2.  Game begins normally with A/B engine making one move, but stalls out after lc0 make's it's move.  It appears frozen and does not make another move with causes the cutechess game to abort.

**Lc0 version**
Both lc0 and lcAB compiled from latest Github source (dirty builds).  

**Cutechess gui parameters**
Screenshot attached

**Hardware**
* GPU: 1 RTX 3060
* 128 GB RAM
* AMD 3960X (48 threads)

**Lc0 logs**
Attached

**Chess GUI logs**
[Termination "stalled connection"]
[TimeControl "30/move"]

1. Nxe4 {+0.49/22 13s} d5 {-0.48/21 13s}
2. Bd3 {+0.47/25 6.4s, Black's connection stalls} 1-0
[logABengine.txt](https://github.com/hans-ekbrand/lc0/files/7697240/logABengine.txt)
[engines - Copy.txt](https://github.com/hans-ekbrand/lc0/files/7697242/engines.-.Copy.txt)
![Screenshot (2)](https://user-images.githubusercontent.com/11835229/145681188-c796d3b8-a028-4d8d-b638-259534b3f81f.png)

**Comment by whelanh** [comment]
FYI, the latest commit bc954d0 did not change the stalling behavior in Windows cutechess gui.  Also, if I leave the AuxEngineFile parameter blank (i.e., do not use a helper engine), the compiled Lc0A/B helper engine runs fine with no issues.  So it's definitely something involving the interaction of the helper engine and lc0.  I am wondering if the helper engine is not getting terminated cleanly after a move is made in cutechess, thus  perhaps leaving Lc0A/B engine in a wait state (thus cutechess cannot restart it for the next move?).  Just an uneducated guess on my part.  I also tried changing the time control in cutechess gui to just limit nodes per move (instead of time), but I got the same behavior (so I don't think it's a time control issue).
